cells.py

```diff
@@ -53,17 +54,19 @@ def load_all(self):
         if new_height <= self.ScreenHeight:
             self.ScreenHeight = new_height
         else:
-            self.ScreenWidth = int(self.ScreenHeight * (aspect_width / aspect_height))
-        self.scale = self.ScreenWidth / 1200  # 1200x900(4/3) is the base resolution
+            aspect_ratio = aspect_width / aspect_height
+            self.ScreenWidth = int(self.ScreenHeight * (aspect_ratio))
+        self.scale = self.ScreenWidth / 1200
```

Comment:
Does floor division affect the scale when used here?

Reply:
Using floor division would affect the scale but the effect would be minimal